### PR TITLE
weave:0.1.0

### PR DESCRIPTION
**Pull request by leana8959**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: lambda aims to help the composition of lambda abstractions.
Is "lambda" too obvious as a package name ? I can change it if needed.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [ ] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by laurmaedje** [comment]
> Is "lambda" too obvious as a package name ? I can change it if needed.

Given the description "to help the composition of lambda abstractions" is is too canonical indeed.

**Comment by leana8959** [comment]
What about "weave" as it weaving functions together ? It's not traditionally used in this context and is relatively unique.

**Comment by laurmaedje** [comment]
Sure, sounds good!

File: packages/preview/weave/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# weave
+A helper library for chaining lambda abstractions, imitating the `|>` or `.` operator in some
+functional languages.
+
+## Import
+You can import the latest version of this package with:
+```typ
+#import "@preview/lambda:0.1.0": pipe_
```
Comment (laurmaedje) [inline packages/preview/weave/0.1.0/README.md:8]:
```suggestion
#import "@preview/weave:0.1.0": pipe_
```

**Comment by leana8959** [comment]
I just realized that I could add some more explanation on how it's useful, do you mind me turning this into a draft temporarily ?
Sorry for the incovenience.

**Comment by laurmaedje** [comment]
Sure, take your time.

**Comment by leana8959** [comment]
I added two more examples, should be good now :)

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**